tests/test_agent.py

```diff
@@ -5807,3 +5807,47 @@ def test_agent_builtin_tools_runtime_vs_agent_level():
             MCPServerTool(id='example', url='https://mcp.example.com/mcp'),
         ]
     )
+
+
+async def test_message_history():
```

Comment:
Make sure you've run `make install`, and then when you run this test with `uv run pytest <path>`, the snapshots should be filled in automatically.

Reply:
Managed to make it work. Thanks.